src/main/kotlin/org/jitsi/jibri/selenium/JibriSelenium.kt

```diff
@@ -131,7 +131,6 @@ class JibriSelenium(
                 "--start-maximized",
                 "--kiosk",
                 "--enabled",
-                "--disable-infobars",
```

Comment:
Is this required to remove?  If we are trying to support both older and newer Chrome versions with this jibri build, we may want to continue to support the old command line parameters at least for some time.

Reply:
yeah i'm tempted to say this should stay in as well for that reason.  it doesn't break anything in chrome 76, does it @netaskd ?

Reply:
you are right. It can be stayed without any problems in chrome 76